src/app/services/weather.service.ts

```diff
@@ -100,6 +100,25 @@ export class WeatherService {
             this.updateLoading(false);
           }
         });
+    if (!this.loading && lat && long) {
+      this.updateLoading(true);
+      this.httpSubscription = this.http.get(`${this.serverURL}?point=44.0805%2C-103.2310&limit=500`)//got rid of ${lat} and long
```

Comment:
this needs fixed - lat/long is hardcoded

Reply:
damn it kerry